src/OrchardCore.Modules/OrchardCore.Search.Lucene/Migrations.cs

```diff
@@ -174,28 +174,31 @@ public int Create()
                             {
                                 logger.LogDebug("Updating Lucene indices settings and queries");
                             }
+                            var quotedTableName = dialect.QuoteForTableName(table, session.Store.Configuration.Schema);
+                            var quotedContentColumnName = dialect.QuoteForColumnName("Content");
+                            var quotedTypeColumnName = dialect.QuoteForColumnName("Type");
 
-                            var updateCmd = $"UPDATE {dialect.QuoteForTableName(table, session.Store.Configuration.Schema)} SET Content = REPLACE(content, '\"$type\":\"OrchardCore.Lucene.LuceneQuery, OrchardCore.Lucene\"', '\"$type\":\"OrchardCore.Search.Lucene.LuceneQuery, OrchardCore.Search.Lucene\"') WHERE [Type] = 'OrchardCore.Queries.Services.QueriesDocument, OrchardCore.Queries'";
+                            var updateCmd = $"UPDATE {quotedTableName} SET {quotedContentColumnName} = REPLACE({quotedContentColumnName}, '\"$type\":\"OrchardCore.Lucene.LuceneQuery, OrchardCore.Lucene\"', '\"$type\":\"OrchardCore.Search.Lucene.LuceneQuery, OrchardCore.Search.Lucene\"') WHERE {quotedTypeColumnName}  = 'OrchardCore.Queries.Services.QueriesDocument, OrchardCore.Queries'";
```

Comment:
Why did `Content` change to quotedContentColumnName?

Reply:
It looks like you have to use double quotes around column names in pgsql


![image](https://user-images.githubusercontent.com/15613121/229152986-8c0ad0d5-b8b6-42a6-882e-e54c3bf239b0.png)

Reply:
you can try this script 

``` sql
UPDATE "Document"
SET Content = REPLACE(Content, '\"$type\":\"OrchardCore.Lucene.LuceneQuery, OrchardCore.Lucene\"', '\"$type\":\"OrchardCore.Search.Lucene.LuceneQuery, OrchardCore.Search.Lucene\"') 
WHERE Type  = 'OrchardCore.Queries.Services.QueriesDocument, OrchardCore.Queries'
```